src/libasr/codegen/wasm_to_x64.cpp

```diff
@@ -50,14 +50,8 @@ class X64Visitor : public WASMDecoder<X64Visitor>,
     void call_imported_function(uint32_t func_idx) {
         switch (func_idx) {
             case 0: {  // print_i32
-                std::cerr << "Call to print_i32() will be printed as exit code\n";
-
-                // Currently, for print we are setting the value to be printed
-                // as the exit code. Later we can access/view this value from console
-                // using: echo $?
-                m_a.asm_pop_r64(X64Reg::rdi); // get exit code from stack top
-                m_a.asm_mov_r64_imm64(X64Reg::rax, 60); // sys_exit
-                m_a.asm_syscall(); // syscall
+                m_a.asm_call_label("print_i64");
+                m_a.asm_pop_r64(X64Reg::r15); // pop the passed argument
```

Comment:
After this is merged, let's document what the "calling convention" is. It looks like the caller is responsible to put arguments on stack, as well as removing them from stack once the called function (`print_i64` in this case) finishes?

Reply:
Yes, the caller is responsible to put arguments on stack. 

In `wasm`, the arguments are already used up (that is there is not need to externally remove them) by the called function.

In assembly `x86` and/or `x64`, we are currently manually/explicitly removing the passed arguments in the `visit_Call()` function:
https://github.com/lcompilers/lpython/blob/7aaedcc9ff91994d8846d9810b35ec7fc8bdffda/src/libasr/codegen/wasm_to_x64.cpp#L97-L123

Reply:
For the `print_i64`, as it is an imported function and as this is an initial support, we are removing the passed arguments just after the call:

https://github.com/lcompilers/lpython/blob/7aaedcc9ff91994d8846d9810b35ec7fc8bdffda/src/libasr/codegen/wasm_to_x64.cpp#L50-L56

Reply:
I think we should follow the WASM convention and the function should remove them, so in this case `print_i64` should remove the arguments from stack. What do you think?